packages/sails-ng-common/src/validation/form.model.ts

```diff
@@ -37,14 +43,46 @@ export type FormValidatorControl = {
    * @param errors The complete map of validation errors.
    */
   setErrors(errors: FormValidatorErrors | null): void;
-};
+}
+
+/**
+ * A simple form validator control that can be used on the server-side to mimic client-side controls.
+ *
+ * Do not use this on the client-side.
+ */
+export class SimpleServerFormValidatorControl implements FormValidatorControl {
+    value: unknown;
+    private _setErrors: (FormValidatorErrors | null)[] = [];
+
+    constructor(value: unknown) {
+        this.value = value;
+    }
+
+    get<P extends string>(path: P): FormValidatorControl | null {
+        const result = _get(this.value, path) ?? null;
+        console.debug(`SimpleServerFormValidatorControl.get path '${path}' with result '${JSON.stringify(result)}' from value '${JSON.stringify(this.value)}'`);
+        return result;
+    }
+    setErrors(errors: FormValidatorErrors | null): void {
```

Comment:
Should this be "setErrors" or "addErrors"?

Reply:
The interface `FormValidatorControl` is intended to be similar to the angular `AbstractControl`, which only has [setErrors](https://angular.dev/api/forms/AbstractControl#setErrors), there's no `addErrors`.

Reply:
The `setErrors` method wasn't used in the end, so I've removed it.